rbc/src/main/java/com/library/rbc/model/enums/SuggestionStatus.java

```diff
@@ -0,0 +1,10 @@
+package com.library.rbc.model.enums;
+
+public enum SuggestionStatus {
+    ORDERED,
+    PENDING,
+    IN_LIBRARY,
+    CANCELED,
+    APPROVED,
+    DENIED
```

Comment:
Minor suggestion: change the order so that it reflects the natural transition of states more closely. 
```
PENDING,
APPROVED,
DENIED,
ORDERED,
CANCELED,
IN_LIBRARY
```

Reply:
On figma design, after the book is ordered it cannot be canceled anymore. Maybe it would be better for status CANCELED to go before status ORDERED.

Reply:
Yeah, makes sense.